docs/runbooks/SSPCommonTemplatesModificationReverted.md

```diff
@@ -0,0 +1,21 @@
+# SSPCommonTemplatesModificationReverted
+
+## Meaning
+
+This alert indicates that one or more common templates were changed by the ssp operator as part of its reconciliation procedure.
+If the operator was upgraded this alert just indicates the common templates were updated to the latest version.
+
+## Impact
+
+If any changes were made to a common template manually or by a script they were reverted by the operator.
+
+## Diagnosis
+
+If the operator was upgraded this alert is to be expected.
+Otherwise the common templates should never be modified manually, find out if you are misusing the common templates by making suc modifications.
+ 
+## Mitigation
+
+Do not modify common templates. If necessary, copy the ones you need and modify the copy.
```

Comment:
@borod108 
Issue with singular vs plural.
Do not modify common templates. If necessary, copy the ones you need and modify the copies.
or
Do not modify a common template. If necessary, copy the one you need and modify the copy.

I think the singular is better.

Reply:
sure, going with the singular.